test/graphql/user/user_query_test.rb

```diff
@@ -0,0 +1,115 @@
+# frozen_string_literal: true
+
+require("test_helper")
+
+module Queries
+  class UserQueryTest < IntegrationTestCase
+    # def setup
+    #   context = {
+    #     session_user: context[:session_user]
+    #   }
+    # end
+
+    def test_find_user_by_id
+      query_string = <<-GRAPHQL
+        query($id: Int){
+          user(id: $id) {
+            name
+            id
+            email
+            login
+          }
+        }
+      GRAPHQL
+
+      user = rolf
+
+      result = MushroomObserverSchema.execute(
+        query_string, variables: { id: user.id }
+      )
+      user_result = result["data"]["user"]
```

Comment:
Is there any way that someone who is not in admin mode could discover the email through this query?

Reply:
Yes - hahaha! This is not really over your head is it Joe?! 

As of this PR nothing is exposed to the internet by the graphql api, but you're right on. Field authorization (more granular than, but in addition to, query and mutation auth) is one of the main things we need to build in on the Rails side. 

Next PR will introduce protected fields, and we'll need to implement them on User. Or if you guys prefer i can add them to this PR once I figure it out. It's pretty simple.